### PR TITLE
[MNO-695] Api v2 Dashboard missing dependency for KPI Alert Recipients

### PR DESCRIPTION
**Pull request by x4d3**
- Include Recipients in Alerts
- Use update_recipients instead of recipient_id

### CODE REVIEW

File: api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/kpis_controller.rb [hunk 1]
```diff
@@ -39,11 +39,11 @@ def index
         kpi.merge(
           name: "#{kpi[:name]} #{watchable.capitalize unless kpi[:name].downcase.index(watchable)}".strip,
           watchables: [watchable],
-          target_placeholders: {watchable => kpi[:target_placeholders][watchable]},
+          target_placeholders: { watchable => kpi[:target_placeholders][watchable] },
         )
       end
     end
-    .flatten
+             .flatten
```
Comment (ouranos) [inline api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/kpis_controller.rb]: ?

File: api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/kpis_controller.rb [hunk 2]
```diff
@@ -26,8 +26,8 @@ def index
 
     begin
       response = MnoEnterprise::ImpacClient.send_get('/api/v2/kpis', attrs, basic_auth: auth)
-      # TODO check there was no error, something like
-      # return render json: { message: "Unable to retrieve kpis from Impac API | Error #{response.code}" } unless response.success?
+        # TODO check there was no error, something like
```
Comment (ouranos) [inline api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/kpis_controller.rb]: ?

File: core/app/models/mno_enterprise/alert.rb [hunk 3]
```diff
@@ -2,5 +2,22 @@ module MnoEnterprise
   class Alert < BaseResource
     property :created_at, type: :time
     property :updated_at, type: :time
+    property :kpi_id
```
Comment (ouranos) [inline core/app/models/mno_enterprise/alert.rb]: Should the `kpi_id` be exposed? I thought we were getting rid of the foreign keys

Comment (x4d3) [reply core/app/models/mno_enterprise/alert.rb]: It should for now, but will be addressed later in https://github.com/maestrano/mno-enterprise/pull/560

File: api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/kpis_controller.rb (hunk 2 above)
Comment (x4d3) [reply api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/kpis_controller.rb]: Rubymine...

**Pull request merged**